sdk/monitor/test-resources.bicep

```diff
@@ -169,6 +172,69 @@ resource dataCollectionRuleRoleAssignment 'Microsoft.Authorization/roleAssignmen
   }
 }
 
+var storageAccountName  = '${uniqueString(utc)}'
+resource storageAccount 'Microsoft.Storage/storageAccounts@2021-09-01' = {
+  name: storageAccountName
+  location: location
+    sku: {
+    name: 'Standard_RAGRS'
+  }
+  kind: 'StorageV2'
+  properties: {
+    defaultToOAuthAuthentication: false
+    allowCrossTenantReplication: true
+    minimumTlsVersion: 'TLS1_2'
+    allowBlobPublicAccess: true
+    allowSharedKeyAccess: true
+    networkAcls: {
+      bypass: 'AzureServices'
+      virtualNetworkRules: []
+      ipRules: []
+      defaultAction: 'Allow'
+    }
+    supportsHttpsTrafficOnly: true
+    encryption: {
+      services: {
+        file: {
+          keyType: 'Account'
+          enabled: true
+        }
+        blob: {
+          keyType: 'Account'
+          enabled: true
+        }
+      }
+      keySource: 'Microsoft.Storage'
+    }
+    accessTier: 'Hot'
+    roleDefinitionId: resourceId('Microsoft.Authorization/roleDefinitions', logReaderRoleId)
```

Comment:
This is awesome, thanks for generating the resources this way :)

Reply:
How are you populating the data in it though

Reply:
I create the Blob in the Storage Account, so the Blob Logs have the records of the creation of the Storage Account already. I don't need to actually populate with data - there will be some already to query.

Reply:
discussed offline